Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml

```diff
@@ -223,6 +217,17 @@ script:
                     }
                 }
             }
+            jsonKeys = Object.keys(jsonRes);
+            if (jsonKeys.length > 20) {
+                limit = 20;
+            } else {
+                limit = jsonKeys.length;
+            }
+            var jsonMd = {};
```

Comment:
just to make sure, we only need to filter the MD? the context itself is not packed?
dose this make sense?

Reply:
The related domains do not go into the context.